config/routes.rb

```diff
@@ -8,7 +8,12 @@
     post 'auto_upload', to: 'uploads#auto_upload'
   end
 
+  namespace :insights_cloud do
+    resources :tasks
```

Comment:
This generates seven routes, but unless I'm missing something `InsightsCloud::TasksController` implements only `#create`